filemanager/filemanager_test.go

```diff
@@ -716,3 +717,105 @@ func TestFileManager_S3(t *testing.T) {
 		}
 	}
 }
+
+func TestS3Manager_SelectObjects(t *testing.T) {
+	minioConfig := map[string]interface{}{
+		"bucketName":       bucket,
+		"accessKeyID":      accessKeyId,
+		"accessKey":        secretAccessKey,
+		"enableSSE":        false,
+		"prefix":           "some-prefix",
+		"endPoint":         minioEndpoint,
+		"s3ForcePathStyle": true,
+		"disableSSL":       true,
+		"region":           region,
+	}
+
+	fm, err := filemanager.New(&filemanager.Settings{
+		Provider: "S3",
+		Config:   minioConfig,
+		Logger:   logger.NOP,
+		Conf:     config.New(),
+	})
+	if err != nil {
+		t.Fatalf("failed to create S3 filemanager: %v", err)
+	}
+
+	parquetPath := "goldenDirectory/test.parquet"
```

Comment:
what is `goldenDirectory`? Is it like `testdata` directory?

Reply:
Yes, there is a [directory](https://github.com/rudderlabs/rudder-go-kit/tree/chore.war-690/filemanager/goldenDirectory) with test files